org.geany.Geany.json

```diff
@@ -0,0 +1,30 @@
+{
+  "app-id": "org.geany.Geany",
+  "runtime": "org.gnome.Sdk",
+  "runtime-version": "3.26",
+  "branch": "stable",
+  "sdk": "org.gnome.Sdk",
+  "command": "geany",
+  "rename-desktop-file": "geany.desktop",
+  "rename-appdata-file": "geany.appdata.xml",
+  "rename-icon": "geany",
+  "finish-args": [
+     "--share=ipc",
+     "--socket=x11",
+     "--share=network",
```

Comment:
I assume you ask for this permission because it uses the network for installing plugins, cloning repos etc.? If so, that's fine. If you don't need it - then remove it.

Reply:
Yeah, I was thinking about plugins, but Git is a much  better reason to do so.